### ISSUE TITLE
lookml-parser 5.0.0 breaks linter & grapher

### ISSUE THREAD

**Issue by iacobus** [ww-tech/lookml-tools#2]
Hi. I believe this commit https://github.com/fabio-looker/node-lookml-parser/commit/ad755f2958c5717cb77971b3bcf3e262d0974382, included in version 5.0.0 of `lookml-parser`, breaks both the linter and the grapher (haven't tried the updater) due to simpler schema of the JSON representation of the LookML files. Apparently they removed the `files` array in favor of a dictionary, calling out the breaking change. With 5.0.0 you get the following error.

```
Traceback (most recent call last):
  File "run_grapher.py", line 42, in <module>
    main()
  File "run_grapher.py", line 39, in main
    LookMlGrapher(config).run()
  File "/Users/jacobo/workspace/lookml-tools/lkmltools/grapher/lookml_grapher.py", line 231, in run
    self.extract_graph_info(globstrings)
  File "/Users/jacobo/workspace/lookml-tools/lkmltools/grapher/lookml_grapher.py", line 220, in extract_graph_info
    self.process_file(filepath)
  File "/Users/jacobo/workspace/lookml-tools/lkmltools/grapher/lookml_grapher.py", line 190, in process_file
    if 'views' in json_data['files'][0]:
KeyError: 'files'
```

Reverting to `lookml-parser` version 4.0.0 resolves this issue.

**Comment by weightwatchers-carlanderson** [comment]
thanks @iacobus. I will look into it.

**Comment by fabio-looker** [comment]
Sorry guys, thought the semver change would be enough to prevent dependency issues. Let me know if there's something I could do to help here.

**Comment by fabio-looker** [comment]
In particular, it might make sense, _for the file/files construct in particular_, to expose an option to choose between object/array. (I would however argue in favor of only the object representation of other constructs within lookml itself)

**Comment by weightwatchers-carlanderson** [comment]
@iacobus I am working through the changes. Have 77/84 tests passing locally now. However, I believe I am now blocked with an issue with the parser itself: https://github.com/fabio-looker/node-lookml-parser/issues/7

**Comment by weightwatchers-carlanderson** [comment]
update: Josh Temple has open sourced a pure Python LookML parser (https://github.com/joshtemple/lkml).

I am currently working to swap out the node parser with this Python one. I now have my test suite passing, so long as I don't have dupe keys in models (https://github.com/joshtemple/lkml/issues/6) or hanging commas (https://github.com/joshtemple/lkml/issues/5).
If I/Josh can fix those issues in the python parser, then I will make a new `lookml-tools` release.

**Comment by weightwatchers-carlanderson** [comment]
@iacobus I've released v2.0.0. This uses a Python LookML parser so that there is no longer a dependency on this node parser.

**Comment by iacobus** [comment]
👍 will report if I encounter trouble. Thanks for the responsiveness @weightwatchers-carlanderson and @fabio-looker.
